.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 Alexander Grund
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at http://boost.org/LICENSE_1_0.txt)
+
+name: Release
+
+on:
+  push:
+    branches: [master]
+    tags: ['v*']
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    env:
+      BOOST_VERSION: 1.71.0
+      BOOST_ROOT: ${{github.workspace}}/dependencies/boost
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
```

Comment:
Why does this need `checkout` twice?

Reply:
The first is this project the 2nd is boost. See the with part below